src/main/controllers/features/unincorporated/whatIsTheBusinessNameController.ts

```diff
@@ -59,14 +59,15 @@ export const post = async (req: Request, res: Response, next: NextFunction) => {
             if (acspData) {
                 acspData.businessName = unincorporatedBusinessName;
                 saveDataInSession(req, USER_DATA, acspData);
-            } else {
-                const acspDataBusinessName : ACSPData = {
-                    businessName: unincorporatedBusinessName
-                };
-                if (session) {
-                    session.setExtraData(USER_DATA, acspDataBusinessName);
-                }
             }
+            //             else {
```

Comment:
remove this commented code actually, it brings code smells

Reply:
addressed